src/megatron/energon/epathlib/epath.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         elif isinstance(initial_path, str):
             protocol, path = self._split_protocol(initial_path)
             if protocol is None:
-                self.internal_path = PurePosixPath(path)
+                self.internal_path = PurePosixPath(PathlibPath(path).absolute())
```

Comment:
I debugged why the tests fail and this turns out to be the problem. This line makes all EPaths absolute, but this is wrong. 
We do need relative EPaths. Those should be allowed as long as it's not an rclone path.

I haven't checked the implications of removing `.absolute()` here. I guess you put it for a reason so it might break other things. Let's discuss this.

Reply:
Oh sorry, my push with that fixed failed. Reverted that change now.